src/modules/temporary-file-check/file.service.ts

```diff
@@ -0,0 +1,25 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { MAX_FILE_SIZE } from '@ukef/constants';
+import { DtfsStorageFileService } from '@ukef/modules/dtfs-storage/dtfs-storage-file.service';
+
+import { GetFileSizeRequestItem } from './dto/get-file-size-request.dto';
+import { GetFileSizeResponse } from './dto/get-file-size-response.dto';
+
+@Injectable()
+export class FileService {
+  constructor(private readonly dtfsStorageFileService: DtfsStorageFileService) {}
+
+  async getFileSize(fileToCheck: GetFileSizeRequestItem): Promise<GetFileSizeResponse> {
+    const { fileName, fileLocationPath } = fileToCheck;
+    const fileSize = await this.dtfsStorageFileService.getFileProperties(fileName, fileLocationPath).then((response) => {
```

Comment:
`fileSizeInBytes` would be a bit more helpful as a variable name

Reply:
Changed